recipe/patches/customize-setup.patch

```diff
@@ -0,0 +1,46 @@
+From 3be2fa4f6b4b85fcd94ba01dd6ac0b5b29f432b8 Mon Sep 17 00:00:00 2001
+From: ngam <67342040+ngam@users.noreply.github.com>
+Date: Fri, 4 Feb 2022 16:16:54 -0500
+Subject: [PATCH] customize setpy
+
+---
+ tensorflow/tools/pip_package/setup.py | 14 +++++++-------
+ 1 file changed, 7 insertions(+), 7 deletions(-)
+
+diff --git a/tensorflow/tools/pip_package/setup.py b/tensorflow/tools/pip_package/setup.py
+index 58de3ad9c40..96043f9719c 100644
+--- a/tensorflow/tools/pip_package/setup.py
++++ b/tensorflow/tools/pip_package/setup.py
+@@ -78,11 +78,11 @@ REQUIRED_PACKAGES = [
+     'google_pasta >= 0.1.1',
+     'h5py >= 2.9.0',
+     'keras_preprocessing >= 1.1.1', # 1.1.0 needs tensorflow==1.7
+-    'libclang >= 9.0.1',
+-    'numpy >= 1.20',
++    # 'libclang >= 9.0.1', # edit ...
++    # 'numpy >= 1.20',  # edit ...
```

Comment:
I don't yet "get" the purpose/intention of this patch. Before it was clear "* Remove unused libclang requirement", but now I don't know what's going on. I still think it should be a rebase of 0004.

Also, what's the problem with a numpy lower bound? Yes, it would be one more thing to override from the global pinning, but that's doable still. I looked upstream for context of this bump, but didn't find any. I [asked](https://github.com/tensorflow/tensorflow/commit/a66666b5ef4baed419ee7d32d1eb60b88486178c), let's see if someone responds.

Reply:
Well, do we really want to go for a lower numpy bound of 1.20 on this? That's my point: I'd personally vote to keep the status quo, hence the `# edit ...` to get people to see it and express their opinions.

I checked with them upstream too and they confirmed this wasn't a mistake (since the title of that PR was talking about placing an  upper bound (language mistake, their response). Now, I think their intention was to add support for numpy 1.20+ and not to actually limit it to 1.20 and above, but they don't want to keep testing, I guess...

Reply:
Yes we want a lower bound. We have to assume that tensorflow knows what they are doing. They might be using new API in one location.

Otherwise, please submit PRs upstream and then we can review patching accordingly. We don't *need* this removal.

Reply:
Alright, that's the point of those obvious edits so that we can agree on a strategy, happy to go for 1.20 and above 😃 

@hmaarrfk I vote to start skipping all but 11.2 for cuda, the 10.2 is failing for some reason... 😢 could you have a look?

Reply:
The décision to drop 10.2 should be independent of releasing this package. Let's keep the discussion separate.

If it becomes a blocker ,pleas create an issue and cross reference

Reply:
Noted.

Reply:
(pinning numpy 1.20.* for py<310 and 1.21.* for py>=310 as py310 didn't exist before 1.21.x)

Reply:
@hmaarrfk, I believe my work is done (sans rebasing the patch) here and this PR work is done. Let's see if anything fails before the timeout (besides the osx-arm64). Then, you may build if you still can and we will see if @xhochy wants me to build or if he wants to do it --- I don't want to build if he ends up rebuilding them anyway. I am happy to build the osx-arm64 ones.

Deferred changes: dropping cuda<11.2 and building tensorflow-io, which can be addressed at a later stage.

Reply:
Whoever has the resources to build can build ;) , I have everything set up but am short on time. Once I get to it, it is simply my machine running for 12h straight and done. At the end, we need someone from core (e.g. me) to copy it to the conda-forge channel.